### PR TITLE
Refactor: my channel claims

### PR DESCRIPTION
**Pull request by saltrafael**
another work originally from `creator-memberships`, makes it better to handle my channels stuff like memberships, premium etc

### CODE REVIEW

**Comment by tzarebczan** [comment]
Is this up on salt for testing too?

File: ui/redux/actions/rewards.js [hunk 1]
```diff
@@ -185,3 +185,9 @@ export function doFetchRewardedContent() {
     Lbryio.call('reward', 'list_featured').then(success, failure);
   };
 }
+
+// @flow
+export const doUserViewRateList = () => async (dispatch: Dispatch) =>
+  await Lbryio.call('user_rewards', 'view_rate').then((data) =>
+    dispatch({ type: ACTIONS.USER_VIEW_RATE_COMPLETED, data })
+  );
```
Comment (infinite-persistence) [inline ui/redux/actions/rewards.js:193]: :small_blue_diamond: Minor:  double call if refreshed on `/$/channels`

image


Probably don't need a full redux flag, but just a local jsx state?

Comment (saltrafael) [reply ui/redux/actions/rewards.js:193]: forgot a `isFetching` f38cb3fbf22ac68aad6f1c821c5da39a544214c7 this should make it 1 call only for each

**Comment by saltrafael** [comment]
now up on https://salt.odysee.com

**Pull request merged**